src/neuroglancer/async_computation/request.ts

```diff
@@ -17,6 +17,10 @@
 import {AsyncComputationSpec} from 'neuroglancer/async_computation';
 import {CANCELED, CancellationToken} from 'neuroglancer/util/cancellation';
 
+declare var BUNDLE_ROOT: string|undefined
+const bundleRoot = "undefined" === typeof BUNDLE_ROOT ? '' : BUNDLE_ROOT;
```

Comment:
I think since this is always loaded from a worker, the path is resolved relative to the worker path, rather than relative to the main document.  Therefore, I think you could just use the existing relative path without the need for any changes.  m I missing something?